src/main/java/org/commcare/core/parse/CommCareTransactionParserFactory.java

```diff
@@ -80,20 +85,31 @@ public TransactionParser getParser(KXmlParser parser) {
             }
             req();
             return userParser.getParser(parser);
+        } else if (FixtureIndexSchemaParser.INDICE_SCHEMA.equalsIgnoreCase(name)) {
+            return new FixtureIndexSchemaParser(parser, fixtureSchemas);
         } else if ("fixture".equalsIgnoreCase(name)) {
+            String id = parser.getAttributeValue(null, "id");
+            String isIndexedAttr = parser.getAttributeValue(null, "indexed");
+            boolean isIndexed = "true".equals(isIndexedAttr);
             req();
-            return fixtureParser.getParser(parser);
+            if (isIndexed) {
+                FixtureIndexSchema schema = fixtureSchemas.get(id);
```

Comment:
Hm, I'm not following what you mean by that I don't think. We can follow up offline